AI/DavinciProxy.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.Azure.CognitiveServices.Language.LUIS.Runtime;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace FFXIVVenues.Veni.AI
+{
+    internal class DavinciProxy
+    {
+        static double maxTokens { get; } = 150;
+        double temperature { get; set; } = 0.7;
+        private readonly string apiKey = "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";//CHANGE TO USE IT ON SECRET
+
+        public String askTheAI (string prompt)
+        {
+            using (var client = new HttpClient())
+            {
+                client.DefaultRequestHeaders.Authorization = new System.Net.Http.Headers.AuthenticationHeaderValue("Bearer", apiKey);
+
+                var requestPayload = new
+                {
+                    prompt = prompt,
+                    temperature = temperature,
+                    max_tokens = maxTokens
+                };
+                var content = new StringContent(JsonSerializer.Serialize(requestPayload), Encoding.UTF8, "application/json");
+                var response = client.PostAsync("https://api.openai.com/v1/engines/text-davinci-003/completions", content).Result;
```

Comment:
We can talk about how to make this asynchronous later. Using `.Result` will lock the current thread until the web request completely. Locking up system resources in a kind of spinwait fashion.

Reply:
Should just able to add `await` to the start of the line. Remove the `.Result`, and add `async` in the signature for this method, just after "public".

Reply:
Oky